Odata-docs/client/v7/batch-operations.md

```diff
@@ -263,3 +260,61 @@ A truncated request Payload will look like this:
 	......
 	......
 ```
+
+## Json Batch
```

Comment:
On this header and the section below, some places you use `Json` and others `JSON`. You should settle on one and use it consistenly

Reply:
#Resolved